### PR TITLE
feat(clerk-js,elements,types,localization): Make legal consent stable

### PR DESCRIPTION
**Pull request by octoper**
## Description

This PR marks legal consent feature as stable

## Checklist

- [X] `npm test` runs as expected.
- [X] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/clerk-js/src/ui/elements/LegalConsentCheckbox.tsx [hunk 1]
```diff
@@ -71,7 +68,7 @@ export const LegalCheckbox = (
       <Flex justify='center'>
         <Field.CheckboxIndicator
           elementDescriptor={descriptors.formFieldCheckboxInput}
-          elementId={descriptors.formFieldInput.setId('__experimental_legalAccepted')}
+          elementId={descriptors.formFieldInput.setId('legalAccepted')}
         />
         <FormLabel
```
Comment (alexcarpenter) [inline packages/clerk-js/src/ui/elements/LegalConsentCheckbox.tsx:73]: Wonder if the checkbox label should dim when the form is submitting similar to the other input labels. It currently stays at full opacity but the checkbox input dims.


https://github.com/user-attachments/assets/333b092f-c206-493c-89a5-bcca0450305b

**Review by alexcarpenter — APPROVED** [review]
Went through and tested different flows with no issues. Nice work.

Only had one comment on the checkbox label not dimming on submission similar to other checkboxes.

File: packages/clerk-js/src/ui/elements/LegalConsentCheckbox.tsx (hunk 1 above)
Comment (octoper) [reply packages/clerk-js/src/ui/elements/LegalConsentCheckbox.tsx]: Fixed!

File: packages/clerk-js/src/core/resources/SignUp.ts [hunk 2]
```diff
@@ -216,7 +210,7 @@ export class SignUp extends BaseResource implements SignUpResource {
     }
 
     const web3Wallet = identifier || this.web3wallet!;
-    await this.create({ web3Wallet, unsafeMetadata, __experimental_legalAccepted: __experimental_legalAccepted });
+    await this.create({ web3Wallet, unsafeMetadata, legalAccepted: legalAccepted });
```
Comment (anagstef) [inline packages/clerk-js/src/core/resources/SignUp.ts:213]:
```suggestion
    await this.create({ web3Wallet, unsafeMetadata, legalAccepted });
```

File: .changeset/twenty-rules-boil.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+'@clerk/localizations': minor
+'@clerk/clerk-js': minor
+'@clerk/elements': minor
+'@clerk/types': minor
+'@clerk/ui': patch
+---
+
+Make legal consent feature stable
```
Comment (anagstef) [inline .changeset/twenty-rules-boil.md:9]:
```suggestion
The Legal consent feature is now stable.

Removed the `__experimental_` preffix.
```

**Review by anagstef — APPROVED** [review]
🔥

**Pull request merged**